src/components/atoms/Icon/Icon.tsx

```diff
@@ -66,7 +66,7 @@ const icons: Record<IconNames, React.ComponentType<any>> = {
   'severity-low': SeverityLow,
   shortcuts: Shortcuts,
   'yaml-syntax': YAMLSyntax,
-  'resource-links': ResourceLinks,
+  'resource-links': ReferenceLinks,
```

Comment:
Can we also change `resource-links` to `reference-links` ?

Reply:
Yes, sure @topliceanurazvan

Reply:
Actually, we do not need referenceLinks as icons, we wanted to change resource nam to reference not to confuse with reources, so I changed all resource-links to reference-links, though it turned out there we had resource-links as well implemented before Tutorial page, so I changed that too, now I have changed it back, will push the changes.

Reply:
Done